Formula/libzdb.rb

```diff
@@ -21,6 +27,11 @@ class Libzdb < Formula
   def install
     system "./configure", "--prefix=#{prefix}", "--disable-dependency-tracking"
     system "make", "install"
+    # The test files that end up in share/libzdb/test/ have @SED@ replaced
+    # with the configure-detected one.  However, install of a homebrew
+    # build that is the build-shim's copy of sed which we definitely don't
+    # want dangling references to!
+    inreplace %w[test/Makefile test/unit test/zdbpp test/select test/pool test/exception], %r{/[^ ]+/sed}, "sed"
```

Comment:
I believe we want to install only *.c/*.cpp files (and don't want to have Makefiles/binaries/*.o files there)

Reply:
Do we? Sometimes I've seen `Makefile`s in test folders that allow you to run tests using `make test`, or something similar. If sanitising the `Makefile` of shim references isn't too complicated I don't see the harm in keeping it... or am I missing something else?

Reply:
> Sometimes I've seen Makefiles in test folders that allow you to run tests using make test, or something similar. 

I don't know a case when homebrew user needs to do it, to be honest. 

> If sanitising the Makefile of shim references isn't too complicated I don't see the harm in keeping it... or am I missing something else?

Any `inreplace` increases maintainers burden (it will require extra attention in case of either file disappears, or content changes and doesn't contain the exact line), so if it doesn't add enough profit it's better not to use it.

And it looks like originally for `libzdb`, we copied this dir, just to use it in test block.

Reply:
That's fair. I don't mind seeing it go then.